corehq/apps/app_manager/app_strings.py

```diff
@@ -313,21 +314,21 @@ def app_strings_parts(self, app, lang, for_default=False, build_profile_id=None)
 
 
 class SelectKnownAppStrings(AppStringsBase):
+    """
+    Like DumpKnownAppStrings, but instead of returning all default
+    translations, only returns those used by the app.
+
+    This is the default behaviour.
+    """
 
     def get_app_translation_keys(self, app):
-        return set.union(set(), *(
-            set(t.keys()) for t in app.translations.values()
-        ))
```

Comment:
:fearful:

Reply:
Any idea what that `set.union(set(), ...)` business was about? Is it meant to create a copy?

Reply:
Set (and dict) comprehensions were introduced in Python 2.7 - this might've been from when we were on 2.6.  Beyond that, I'm guessing this is meant to illustrate that this is the union of the set of keys of each app's translations.